src/generators/integer/index.ts

```diff
@@ -4,13 +4,13 @@ import {Chance} from 'chance';
  * Generates a random integer (could be negative!). Optionally between `lowerExclusive` and `upperExclusive`.
  */
 const createIntegerGenerator = (chance: Chance.Chance) => (
-  lowerInclusive = -100,
-  upperExclusive = 100,
+  lowerInclusive?: number,
+  upperExclusive?: number,
 ): number =>
   chance.integer({
```

Comment:
The idea here is that chance is handling the default min/max behavior? Can we document this in a comment?

Reply:
Addressed in 061f610